Packs/MicrosoftCloudAppSecurity/Integrations/MicrosoftCloudAppSecurity/MicrosoftCloudAppSecurity.py

```diff
@@ -0,0 +1,601 @@
+            readable_output['title'] = alert['title']
+        if alert.get('description'):
+            readable_output['description'] = alert['description']
+        if alert.get('statusValue'):
+            readable_output['status_value'] = [key for key, value in STATUS_OPTIONS.items()
+                                               if alert['statusValue'] == value]
+        if alert.get('severityValue'):
+            readable_output['severity_value'] = [key for key, value in SEVERITY_OPTIONS.items()
+                                                 if alert['severityValue'] == value]
+        list_readable_output.append(readable_output)
+    headers = ['alert_id', 'alert_date', 'title', 'description', 'status_value', 'severity_value']
+    human_readable = tableToMarkdown('Results', list_readable_output, headers, removeNull=True)
+    return human_readable
+
+
+def list_alerts_command(client, args):
+    url_suffix = '/alerts/'
+    alert_id = args.get('alert_id')
+    customer_filters = args.get('customer_filters')
+    arguments = assign_params(skip=args.get('skip'), limit=args.get('limit'), severity=args.get('severity'),
+                              service=args.get('service'), instance=args.get('instance'),
+                              resolution_status=args.get('resolution_status'), username=args.get('username'))
+    request_data, url_suffix = build_filter_and_url_to_search_with(url_suffix, customer_filters, arguments, alert_id)
+    alerts = client.list_alerts(url_suffix, request_data)
+    human_readable = alerts_output_to_readable_output(alerts)
+    return CommandResults(
+        readable_output=human_readable,
+        outputs_prefix='MicrosoftCloudAppSecurity.Alert',
+        outputs_key_field='alert_id',
+        outputs=alerts
+    )
+
+
+def bulk_dismiss_alert_command(client, args):
+    alert_ids = args.get('alert_ids')
+    customer_filters = args.get('customer_filters')
+    comment = args.get('comment')
+    request_data = args_to_filter_dismiss_alerts_and_resolve_alerts(alert_ids, customer_filters, comment)
+    dismiss_alerts = client.dismiss_bulk_alerts(request_data)
+    return CommandResults(
+        readable_output=dismiss_alerts,
+        outputs_prefix='MicrosoftCloudAppSecurity.AlertDismiss',
+        outputs_key_field='alert_ids',
+        outputs=dismiss_alerts
+    )
+
+
+def bulk_resolve_alert_command(client, args):
+    alert_ids = args.get('alert_ids')
+    customer_filters = args.get('customer_filters')
+    comment = args.get('comment')
+    request_data = args_to_filter_dismiss_alerts_and_resolve_alerts(alert_ids, customer_filters, comment)
+    resolve_alerts = client.resolve_bulk_alerts(request_data)
+    return CommandResults(
+        readable_output=resolve_alerts,
+        outputs_prefix='MicrosoftCloudAppSecurity.AlertResolve',
+        outputs_key_field='alert_ids',
+        outputs=resolve_alerts
+    )
+
+
+def activities_output_to_readable_output(activities):
```

Comment:
this function is almost a complete duplicate of the same function for files and activities, with minor changes. let's try to avoid this situation

Reply:
same comments as `alerts_output_to_readable_output`